### ISSUE TITLE
Initialise Multiple Image Groups

### ISSUE THREAD

**Issue by SteveBiggs** [FreeUKGen/MyopicVicar#1600]
Economic 0, Urgency 3, strategy 8, cost ?6? Priority ?17

**Comment by SteveBiggs** [comment]
This is required before we can ask coordinators to start marking their image group status so that the RAP report is accurate.  I would like to ask Bev to work on this before Kirk returns.

**Comment by bev-fc** [comment]
I will work on it.

**Comment by SteveBiggs** [comment]
Thanks Bev. Let me know if you need any more information.

**Comment by bev-fc** [comment]
@SteveBiggs please verify if the change works as expected.

I am not sure with one thing: For "Initialize Image Groups" page, should I include both UNALLOCATED and UNINITIALIZED (the source is not initialized yet) image groups or just UNALLOCATED image groups.

**Comment by SteveBiggs** [comment]
Thanks @bev-fc, yes it works as expected.

It should only be Unallocated that can be set here. Uninitialised can be changed to Unallocated at the Source level.

There is another issue (#1544 ) to allow Completed and Transcribed groups to be changed, but they should only be done at the individual group level since they will be a rare exception.

**Comment by bev-fc** [comment]
@SteveBiggs Modified to only Unallocated groups will be listed

**Comment by SteveBiggs** [comment]
Thanks @bev-fc
